scipy/linalg/tests/test_lapack.py

```diff
@@ -1666,75 +1666,90 @@ def generate_random_dtype_array(shape, dtype):
 
 @pytest.mark.parametrize("size", [(6, 5), (5, 5)])
 @pytest.mark.parametrize('dtype', DTYPES)
-@pytest.mark.parametrize('gejsv_lambda',
-                         [(lambda A, gejsv: gejsv(A)),  # base default case
-                          (lambda A, gejsv: gejsv(A, joba="E")),  # joba
-                          (lambda A, gejsv: gejsv(A, joba="F")),
-                          (lambda A, gejsv: gejsv(A, joba="G")),
-                          (lambda A, gejsv: gejsv(A, joba="A")),
-                          (lambda A, gejsv: gejsv(A, joba="R")),
-                          (lambda A, gejsv: gejsv(A, jobu="F")),  # jobu
-                          (lambda A, gejsv: gejsv(A, jobu="W")),
-                          (lambda A, gejsv: gejsv(A, jobv="J")),  # jobv
-                          (lambda A, gejsv: gejsv(A, jobv="V")),
-                          (lambda A, gejsv: gejsv(A, jobr="N")),  # jobr
-                          (lambda A, gejsv: gejsv(A, jobt="T")),  # jobt
-                          (lambda A, gejsv: gejsv(A, jobp="N")),  # jobp
+@pytest.mark.parametrize('job',
+                         [None,
+                          *zip(["joba"]*5,
+                               ["'E'", "'F'", "'G'", "'A'", "'R'"]),
+                          *zip(["jobu"]*3, ["'F'", "'W'", "'N'"]),
+                          *zip(["jobv"]*3, ["'J'", "'W'", "'N'"]),
+                          ("jobr", "'N'"),
+                          ("jobt", "'T'"),
+                          ("jobp", "'P'"),
+                          [("jobv", "'N'"), ("jobu", "'N'")]
```

Comment:
@mdhaber  I believe this covers all of the options by varying each job* value, since the defaults are checked with the default  gejsv call.

Reply:
I'm sorry; by:
> This is not all the options we need tests for. I might not have addressed them all in the original issue, so let's talk about each possibility.

I didn't mean you needed to test all combinations of options. I meant that some values of the options were missing, I think.

Reply:
Yes, I think we are on the same page about this. 

I originally a couple joba values were missing, now they should all be there. This is the same idea, It varies each individual job* option on their own.